.github/workflows/graalvm-latest.yml

```diff
@@ -34,7 +34,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 6
-      matrix: ${{ fromJson(needs.build_matrix.outputs.matrix) }}
+      matrix:
+        java: ['17', '21']
```

Comment:
Do we need to run graalvm tests against both versions since this would duplicate number of slow native tests for each project but it would be especially painful for projects with a large number of those tests like `micronaut-sql` and `micronaut-data`?

Reply:
I would say probably yes...

We want to be testing Micronaut native compilation against LTS JDK releases, and I guess this includes Graal JDKs

@sdelamo for micronaut-data this will double the number of GraalVM latest actions from 17 (currently)  to 34

Reply:
I think we should